lib/kinesisClient.js

```diff
@@ -7,18 +7,26 @@ const kinesis = new AWS.Kinesis({region : config.KINESIS_REGION});
 
 /**
  * Writes to Kinesis
- * @param {Array} uuids -
+ * @param {Array/String} uuids - takes an Array of uuids Or a single
  * @param {String} event -
  * @param {Array} eventData -
  * @returns {Promise} result -
  */
 function writeToKinesis (uuids, event, eventData) {
-  const operation = 'kinesisClient.writeToKinesis';
-  logger.info({operation, uuids, event});
+  const operation = '😾 kinesisClient.writeToKinesis';
+  logger.info({operation, uuids, event, dontWriteType: typeof config.DONT_WRITE_TO_KINESIS, dontWrite: config.DONT_WRITE_TO_KINESIS, uuidsIaA:  typeof uuids});
 
-  if (config.DONT_WRITE_TO_KINESIS === true) {
+  if(!Array.isArray(uuids) && typeof uuids === 'string') {
+    //syncUserFollows provides a string not array
+    //records expects an array so make it so
+    logger.info({operation, subOp: 'horible mutation so uuids.map works 🙀'});
+    uuids = [uuids];
```

Comment:
Do we want this logged out?
Also, could we avoid mutating by assigning to another variable and using that with `map()`?

Reply:
done